pyfr/integrators/base.py

```diff
@@ -58,6 +58,29 @@ def __init__(self, backend, rallocs, mesh, initsoln, cfg):
         # Abort computation
         self.abort = False
 
+        # Smoothly step to target time in the last near_t steps
+        self.fact = self.cfg.getfloat('solver-time-integrator', 'dt-fact', 0.9)
+        self.fact_max = 1.001
```

Comment:
Shouldn't this be configurable?

Reply:
I will make it configurable. 
I only added it since a value slightly higher than the set `dt` took 2 steps instead of 1 slightly larger step. For stiffer problems we will use a conservative `self.fact_max = 1.000001`.